<?xml version="1.0" encoding="UTF-8"?>

<launch>
<!--211Node-->
	<node pkg="tricat221" type="lidar_converter.py" name="obstacles" respawn="true" output="screen"/>
	<!--<node pkg="i-tricat221"  type="gnss_converter.py" name="" respawn="true"/>
	<node pkg="i-tricat221"  type="heading_calculator.py" name="Compass" respawn="true"/>
	<node pkg="i-tricat221"  type="hopping_tour.py" name="" respawn="true" output="screen"/>-->

<!--PARAMETER-->
	<rosparam command="load" file="$(find tricat221)/params/lidar_params.yaml"/>
	<!--<rosparam command="load" file="$(find tricat221)/params/coordinates.yaml"/>
	<rosparam command="load" file="$(find tricat221)/params/hopping_params.yaml"/>-->

<!--LiDAR-->
	<node name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
	    <param name="serial_port"         type="string" value="/dev/ttyLiDAR"/>  
	    <param name="serial_baudrate"     type="int"    value="256000"/>
	    <param name="frame_id"            type="string" value="laser"/>
	    <param name="inverted"            type="bool"   value="false"/>
	    <param name="angle_compensate"    type="bool"   value="true"/>
	    <param name="scan_mode"           type="string" value="Stability"/>
	</node>
	<!--<node name="rviz" pkg="rviz" type="rviz"/> args="-d $(find rplidar_ros)/rviz/rplidar.rviz" />-->

<!--GPS-->
	<!--<include file="$(find ublox_gps)/launch/ublox_device.launch">-->
	    <!--<arg name="node_name" value="ublox_gps"/>
	    <arg name="param_file_name" value="zed-f9p"/>
	    <arg name="output" value="screen"/>
	    <arg name="respawn" value="true"/>
	    <arg name="respawn_delay" value="30"/>
	    <arg name="clear_params" value="false"/>-->
	<!-- </include> Check ublox config file tty -->

	<!--<node pkg="ntrip_ros" type="ntripclient.py" name="ntrip_ros">-->
	    <!--<param name="rtcm_topic" value="/ublox_gps/rtcm"/> Check ublox_gps scripts-->
	    <!--<param name="ntrip_server" value="vrs3.ngii.go.kr:2101"/>
	    <param name="ntrip_user" value="717lumos"/>
	    <param name="ntrip_pass" value="ngii"/>
	    <param name="ntrip_stream" value="VRS-RTCM31"/>
	    <param name="nmea_gga" value="$GPGGA,024539.902,3727.004,N,12639.255,E,1,12,1.0,0.0,M,0.0,M,,*60"/>
	</node>-->

<!--IMU-->
	<!--<node pkg="myahrs_driver" type="myahrs_driver" name="myahrs_driver">
  	    <param name="port" value="/dev/ttyIMU" />
	    <param name="baud_rate" value="115200" />
 	</node>-->

<!--Arduino-->
	<!--<node pkg="tricat_211" type="serial_node.py" name="ARDUINO" args="/dev/ttyARDUINO" respawn="true"/>-->

<!--RQT graph-->
	<!--<node name="rqt" pkg="rqt_graph" type="rqt_graph"/>-->
</launch>
